components/overflow-group/test/overflow-group.visual-diff.js

```diff
@@ -12,6 +12,7 @@ describe('d2l-overflow-group', () => {
 		browser = await puppeteer.launch();
 		page = await visualDiff.createPage(browser);
 		await page.goto(`${visualDiff.getBaseUrl()}/components/overflow-group/test/overflow-group.visual-diff.html`, { waitUntil: ['networkidle0', 'load'] });
+		await page.waitForTimeout(500);
```

Comment:
Could you throw a comment next to this just explaining that it's to avoid flake with the icons not being loaded? Thinking about future-us seeing this and being like "hmm.... delete delete delete".

Reply:
Yeah, added a comment! This does seem ripe for deleting without realizing what its doing.